packages/excalidraw/types.ts

```diff
@@ -443,7 +438,10 @@ export interface ExcalidrawProps {
   UIOptions?: Partial<UIOptions>;
   detectScroll?: boolean;
   handleKeyboardGlobally?: boolean;
-  onLibraryChange?: (libraryItems: LibraryItems) => void | Promise<any>;
+  onLibraryChange?: (
+    libraryItems: LibraryItems,
+    change: LibraryChange,
+  ) => void | Promise<any>;
```

Comment:
ideally I'd like to migrate to `onLibraryChange(data)` where `data` will contain `libraryItems` and `change`, but I'm not sure how to go about this API change as people will not read breaking changes and keep persisting to DB whatever we were passing in to `onLibraryChange` previously.

Reply:
Yeah, we could also separate to a standalone incremental endpoint (`onChangeIncremental`) for those interested in that level of granularity. Right now it however does not make as much sense for libraries as it does for scenes.

Plus once you persist changes you also need to think about migrating them, we should likely provide tooling to run these migrations on the level of changes.

Reply:
I wouldn't expose this through the public API (or marked as `@expermiental` at least) until we cover migrations & proceed further with #7348.

Reply:
> I wouldn't expose this through the public API (or marked as @expermiental at least) until we cover migrations & proceed further with https://github.com/excalidraw/excalidraw/pull/7348.

Fair enough. We can keep this all private to the `useLibraryHandle` for now. I'll remove the `change` params, probably even the `onLibraryChange` emitter until we settle on public API and remove the deprecations.